inspectors/models/notation/sectionbreaks/sectionbreaksettingsmodel.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SECTIONBREAKSETTINGSMODEL_H
+#define SECTIONBREAKSETTINGSMODEL_H
+
+#include "models/abstractinspectormodel.h"
+
+class SectionBreakSettingsModel : public AbstractInspectorModel
+{
+    Q_OBJECT
+
+    Q_PROPERTY(PropertyItem* startWithLongInstrNames READ startWithLongInstrNames CONSTANT)
+    Q_PROPERTY(PropertyItem* resetBarNums READ resetBarNums CONSTANT)
+    Q_PROPERTY(PropertyItem* pause READ pause CONSTANT)
+
+public:
+    explicit SectionBreakSettingsModel(QObject* parent, IElementRepositoryService* repository);
+
+    void createProperties() override;
+    void requestElements() override;
+    void loadProperties() override;
+    void resetProperties() override;
+
+    PropertyItem* startWithLongInstrNames() const;
+    PropertyItem* resetBarNums() const;
+    PropertyItem* pause() const;
+
+private:
+    PropertyItem* m_startWithLongInstrNames = nullptr;
+    PropertyItem* m_resetBarNums = nullptr;
+    PropertyItem* m_pause = nullptr;
```

Comment:
In the new code style, just underscore is needed for class members 
https://github.com/musescore/Documentation/pull/4

Reply:
Isn't `m_` the standard prefix for Qml-readable properties?

Reply:
@Howard-C Hey.
No, this is not related to the properties of Qml, these fields should be considered as simply members of the class.  
i.e. properties getters of pp can be very different

```
Q_PROPERTY(QString myString READ myString ...
...
QString myString() const 
{
   return m_myString;
}

OR

QString myString() const 
{
   return object()->someString();
}

OR 

QString myString() const 
{
   return _muStruct.someString;
}

...
```

Reply:
Oh, interesting... But since `m_` is a convention, it can make the code clear :)

Reply:
Yo, we have not yet approved a new code style among ourselves. I prefer m_member, because underlined names are often used locally in methods or lambdas, in case of name conflict

Reply:
Convenience of `m_` is that it is customary for Qt, and QtCreator automatically generates a property code with this prefix (Alt+Enter, Enter)    

But __most__ of the members of the classes that the developer writes are not related to this.
It’s definitely not necessary to make different prefixes for regular class members and members of the Qml property class

Reply:
> Yo, we have not yet approved a new code style among ourselves. I prefer m_member, because underlined names are often used locally in methods or lambdas, in case of name conflict
> 
> I probably skipped this point when I reviewed you

Please write about this in the PR style code
https://github.com/musescore/Documentation/pull/4

Reply:
Convinced me let's use `m_`